features/step_definitions/create_menu_page.rb

```diff
@@ -16,8 +16,17 @@
       path = new_user_registration_path
     when 'restaurants' then
       path = new_restaurant_path
+    when 'create dish' then
```

Comment:
Is this really necessary? You have a step that visits the create dish page and the path is set there.